### ISSUE TITLE
Fishtank LIVE | fishtank.live

### ISSUE THREAD

**Issue by HQv2** [PreMiD/Activities#7799]
### Website name

Fishtank LIVE

### Website URL

https://www.fishtank.live/

### Website logo

https://upload.wikimedia.org/wikipedia/commons/5/5a/Fishtanklogo.png

### Prerequisites

- [X] It is a paid service
- [ ] It displays NSFW content
- [X] It is region restricted

### Description

Fishtank LIVE is an interactive reality competition webseries created by Sam Hyde and Jet Neptune. During its run, Fishtank is streamed online 24/7, allowing viewers to interact with contestants and the house they reside at in real time. Taking place over 42 days (6 weeks), Fishtank confines 8 contestants in a house and puts them through irregularly-scheduled challenges, eliminating contestants until only one remains as the winner.

The Fishtank is a streamer house monitored 24/7 and broadcasted live to the world. All bedrooms have cameras, common spaces are monitored via video and audio surveillance. When the lights go out, the IR comes on. The Fishtank house is always on.

**Season 2 begins on December 18th, 2023.**

**Features requested:**
-Display current level/rank
-Which live camera you are watching
-Which on-demand video you are watching
-Time elapsed (time spent browsing the website)

**Comment by veryCrunchy** [comment]
@theusaf 
https://github.com/PreMiD/Presences/labels/paid%20service & https://github.com/PreMiD/Presences/labels/nsfw

**Comment by HQv2** [comment]
You can actually view all the live videos without paying, there will just be a few features locked such as chatting.

**Comment by veryCrunchy** [comment]
Can't view season 1 without paying, correct?

**Comment by HQv2** [comment]
> Can't view season 1 without paying, correct?

Correct. The live cams are supposed to start today around 1:30PM EST.
